.github/workflows/ci.yml

```diff
@@ -136,4 +114,4 @@ jobs:
           PGP_SECRET: ${{ secrets.PGP_SECRET }}
           SONATYPE_PASSWORD: ${{ secrets.SONATYPE_PASSWORD }}
           SONATYPE_USERNAME: ${{ secrets.SONATYPE_USERNAME }}
-        run: sbt ++${{ matrix.scala }} ci-release
+        run: sbt ci-release
```

Comment:
@sloshy , do you have any insights into whether this change will break things? It's the only one in this MR that looks a bit fishy to me.